### ISSUE TITLE
Implement batch editing

### ISSUE THREAD

**Issue by bp323** [CUL-DigitalServices/grasshopper-ui#114]
Implement batch editing per design. This depends on #112. We should split this out into multiple smaller tasks to make the whole process easier to manage.

- [x] Create a 'List' and 'Calendar' tab where list should be implemented as batch editing. Calendar can be empty for now
- [x] Create the batch editing header
- [x] Implement the 'add event' functionality https://github.com/CUL-DigitalServices/grasshopper-ui/pull/140
- [x] Create a partial for a single event row https://github.com/CUL-DigitalServices/grasshopper-ui/pull/140
- [x] Hook up the batch edit page to the modules in the sidebar https://github.com/CUL-DigitalServices/grasshopper-ui/issues/127
- [x] Implement jEditable (or similar if better alternatives are available) for the event title, lecturer and room single event edits
- [x] Hook up jEditable fields to batch editing header (title, lecturer, room)
- [x] Create the time/dates batch editing dropdown
![screen shot 2015-01-16 at 14 59 43](https://cloud.githubusercontent.com/assets/218391/5778085/8338a7a6-9d90-11e4-8733-05502248ffed.png)

- [x] Implement the date and time popover for single event edits https://github.com/CUL-DigitalServices/grasshopper-ui/issues/146
![screen shot 2015-01-16 at 14 56 42](https://cloud.githubusercontent.com/assets/218391/5778032/e699097c-9d8f-11e4-8529-fd27fa80da3b.png)

- [x] Implement the `type of event` dropdown editor for single event edits https://github.com/CUL-DigitalServices/grasshopper-ui/issues/139
- [x] Implement the series settings cogwheel button in the right top corner https://github.com/CUL-DigitalServices/grasshopper-ui/pull/137
- [x] Implement the `calendar icon` and date visualisation next to the event date https://github.com/CUL-DigitalServices/grasshopper-ui/pull/138

**Comment by bp323** [comment]
Assigning to myself for implementation

**Comment by bp323** [comment]
Created a separate issue for `Hook up the batch edit page to the modules in the sidebar` at https://github.com/CUL-DigitalServices/grasshopper-ui/issues/127

**Comment by bp323** [comment]
PR for settings cog at https://github.com/CUL-DigitalServices/grasshopper-ui/pull/137

**Comment by Coenego** [comment]
PR for calendar icon at https://github.com/CUL-DigitalServices/grasshopper-ui/pull/138
PR for date and time popover at https://github.com/CUL-DigitalServices/grasshopper-ui/pull/133

**Comment by bp323** [comment]
PR for adding events at https://github.com/CUL-DigitalServices/grasshopper-ui/pull/140

**Comment by bp323** [comment]
Closing as all main issues have been addressed
